docs/security/self-assessment.md

```diff
@@ -0,0 +1,188 @@
+
+## Security issue resolution
+
+### Responsible Disclosures Process
+
+KubeEdge project vulnerability handling related processes are recorded in [Security Policy](https://github.com/kubeedge/kubeedge/security/policy), Related security vulnerabilities can be reported and communicated via email `cncf-kubeedge-security@lists.cncf.io`.
+
+### Incident Response
+
+See the [KubeEdge releases page](https://github.com/kubeedge/kubeedge/releases) for information on supported versions of KubeEdge. Once the fix is confirmed, the Security Team will patch the vulnerability in the next patch or minor release, and backport a patch release into the latest three minor releases.
+
+The release of low to medium severity bug fixes will include the fix details in the patch release notes. Any public announcements sent for these fixes will be linked to the release notes.
+
+## Appendix
+
+### Known Issues Over Time
+
+No known vulnerabilities have been reported.
+
+### [CII Best Practices](https://www.coreinfrastructure.org/programs/best-practices-program/)
+
+KubeEdge has achieved an Open Source Security Foundation (OpenSSF) best practices badge at `passing` level, see more details at [KubeEdge's openssf best practices](https://bestpractices.coreinfrastructure.org/en/projects/3018).
+
+### Case Studies
+
+KubeEdge has been widely adopted in industries including transportation, energy, Internet, CDN, manufacturing, smart campus etc. 
+
+- Xinghai IoT is an IoT company that provides comprehensive smart building solutions by leveraging a construction IoT platform, intelligent hardware, and AI. Xinghai IoT built a smart campus with cloud-edge-device synergy based on KubeEdge and its own Xinghai IoT cloud platform, greatly improving the efficiency of campus management. With AI assistance, nearly 30% of the repetitive work is automated. In the future, Xinghai IoT will continue to collaborate with KubeEdge to launch KubeEdge-based smart campus solutions.
+
+
+* In China’s highway electronic toll collection (ETC) system, KubeEdge helps manage nearly 100,000 edge nodes and more than 500,000 edge applications in 29 of China’s 34 provinces, cities and autonomous regions. With these applications, the system processes more than 300 million data records daily and has improved traffic efficiency at the toll stations by 10 times.
+* China Mobile On-line Marketing Service Center, is a secondary organ of the China Mobile Communications Group, holds the world’s largest call center with 44,000 agents, 900 million users, and 53 customer service centers, builds a cloud-edge synergy architecture consisting of two centers and multiple edges based on KubeEdge. See more details [here](https://www.cncf.io/blog/2021/08/16/china-mobile-kubeedge-based-customer-service-platform-featuring-edge-cloud-synergy).

```

Comment:
[FYI](https://vmblog.com/archive/2021/04/16/managing-100-000-edge-nodes-on-china-s-highways-using-kubeedge.aspx#.YmfoS-dByUk),[FYI](https://www.cncf.io/blog/2021/01/25/maintainer-spotlight-kevin-wang-of-kubeedge-and-volcano/)